fairworkflows/rdf_wrapper.py

```diff
@@ -75,6 +75,18 @@ def remove_attribute(self, predicate, object=None):
         """
         self._rdf.remove((self.self_ref, predicate, object))
 
+    def shacl_validate(self, shaclfile="./shacl/plex-shapes.ttl"):
+        sg = rdflib.Graph()
+        sg.parse(shaclfile, format='ttl')
+        r = pyshacl.validate(self._rdf, shacl_graph=sg, inference='rdfs', abort_on_error=False, meta_shacl=False, advanced=False, js=False, debug=False)
+        conforms, results_graph, results_text = r
+
+        if not conforms:
+            print(results_graph)
```

Comment:
I would prefer something different than printing here.
Maybe raise an error or warning with the results? Or maybe an assertion like in 'validate()' method? (would be nice to have consistent behavior of these methods)

Reply:
Agreed. I will make the behaviour look more like the current ```validate()``` method's.

Reply:
Done.